src/EditorFeatures/Test/Completion/CompletionServiceTests.cs

```diff
@@ -47,6 +47,13 @@ class Test {
             var caretPosition = workspace.DocumentWithCursor.CursorPosition ?? throw new InvalidOperationException();
             var completions = await completionService.GetCompletionsAsync(document, caretPosition, CompletionOptions.Default, OptionValueSet.Empty);
 
+            // NuGet providers are not included until it's loaded and cached, this is to avoid potetnial delays, especially on UI thread.
```

Comment:
```suggestion
            // NuGet providers are not included until it's loaded and cached, this is to avoid potential delays, especially on UI thread.
```